zshrc

```diff
@@ -79,7 +79,7 @@ alias youtube-dl="command youtube-dl --no-mtime"
 # Pipe to this to quote filenames with spaces
 alias quote="sed 's/.*/\"&\"/'"
 alias epoch="date -r"
-alias ag="ag --width 100"
+alias rg="command rg --ignore-file ~/.searchignore --max-columns 200"
```

Comment:
small thing I learned as a part of ignoring `vim-plug` in my search results that you might already know: both `ag` and `rg` respect `.ignore` files, so if you use `.ignore` instead of project specific ones you can bounce back and forth between search tools at will without needing to change all that crap.